airflow/migrations/versions/0017_2_9_2_fix_inconsistency_between_ORM_and_migration_files.py

```diff
@@ -236,6 +236,9 @@ def upgrade():
                 log_template_id INTEGER,
                 updated_at TIMESTAMP,
                 clear_number INTEGER DEFAULT '0' NOT NULL,
+                scheduled_by_job_id INTEGER,
+                context_carrier VARCHAR(50),
+                span_status VARCHAR(50),
```

Comment:
If we need to add new columns then we need to add a new migration file, not edit an existing one.

Is 50 chars enough for the context_carrier? It seems on the short side.

Reply:
I didn't know that. I realized that this was an issue while running the tests. Is there somewhere any guidelines about the migration test?

Reply:
I tried running

```shell
> breeze generate-migration-file -m "add_new_otel_span_fields"
...
...
[2024-12-15T12:56:10.826+0000] {migration.py:207} INFO - Context impl SQLiteImpl.
[2024-12-15T12:56:10.827+0000] {migration.py:210} INFO - Will assume non-transactional DDL.
/opt/airflow/airflow/migrations/env.py:133 SAWarning: WARNING: SQL-parsed foreign key constraint '('dag_id', 'dag', 'dag_id')' could not be located in PRAGMA foreign_keys for table dag_schedule_asset_alias_reference
No change detected in ORM schema, skipping revision.
```

What am I missing?